### ISSUE TITLE
Allow setup during login

### ISSUE THREAD

**Issue by ChristophWurst** [nextcloud/twofactor_totp#574]
https://github.com/nextcloud/twofactor_totp/pull/550

**Comment by xtrasimplicity** [comment]
@ChristophWurst I note that this has been moved to "out of scope", but that the relevant PR (#550) had been merged.

Is this functionality still outstanding from the latest release? If so, what do we need to do to get it into the next release? Happy to contribute some time to help get this sorted. Thanks! 👍

**Comment by ChristophWurst** [comment]
Hi,

you're right. This is a bit confusing. What I meant with "out of scope" on that project was to declare it as not a mandatory part of the feature that has to be done before the Nextcloud 17 feature freeze. It will be released for Nextcloud 17, though :v: 

I'll try to prepare a release this week and will ping you for testing. Thanks a lot for your help :)

**Comment by xtrasimplicity** [comment]
> Hi,
> 
> you're right. This is a bit confusing. What I meant with "out of scope" on that project was to declare it as not a mandatory part of the feature that has to be done before the Nextcloud 17 feature freeze. It will be released for Nextcloud 17, though 
> 
> I'll try to prepare a release this week and will ping you for testing. Thanks a lot for your help :)

Brilliant, thanks, @ChristophWurst! I look forward to testing it. :)
